### DISCUSSION TITLE
files.associations

### DISCUSSION THREAD

**Discussion by rdela** [mvsde/vscode-webc#2] [General]
Thanks for the great extension! I already had: 

```json
"files.associations": {
		"*.webc": "html",
```

in `settings.json`, I should leave that there right? That is still necessary? Do you think this is worth documenting?

**Comment by mvsde** [comment]
Thank you for checking out the extension 🙌

The WebC file to HTML association is configured as a default in this extension, so you can remove it from your VS Code settings as long as you have this extension installed.

The relevant code can be found in the `package.json` of this extension:

https://github.com/mvsde/vscode-webc/blob/10c68aae07800252a37e0b89c7a3c9116129452b/package.json#L40-L44
